deploy-board/deploy_board/templates/groups/group_details.html

```diff
@@ -339,6 +339,24 @@ <h4 class="modal-title">Disable Confirm</h4>
   </div><!-- /.modal-dialog -->
 </div><!-- /.modal -->
 
+<div class="modal fade" id="launchWarningModalId" tabindex="-1" role="dialog"
+     aria-labelledby="launchWarningModalLabel" aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
+        <h4 class="modal-title">Cannot Launch New Host</h4>
+      </div>
+      <div class="modal-body">
+        <strong>Warning!</strong> An instance cannot be launched. Please update the <a href="https://deploy.pinadmin.com/groups/{{ groupName }}/config/">Group Launch Configuration</a> before launching hosts! Please make sure to fill out the IAM role and subnets. 
```

Comment:
Can we check groupInfo to see which field is missing and make the warning message more accurate?